### PR TITLE
feat(howitworks.jsx): designing a section component

### LINKED ISSUE #21: Design features section component
**Task title**
Design a features section component 

**Task description**
Design a compact features section component that showcases key product or service attributes. This component will allow users to quickly grasp the main benefits and functionalities in a visually appealing and easily scannable format.

**Screenshots**
image

[**Link to the component on Figma**
](https://www.figma.com/file/FYZIsFvg1rLu50UnriUN8Y/Student-Store-Project?type=design&node-id=0-1&mode=design&t=0DdQfVRHQ85SzzLb-0)

### PR DESCRIPTION
**Pull request by Bolphunga**
Changes made:
Added a section explaining how the website works.

Screenshot:

Related issue:
Resolves #21 

# Related Issue

- Resolve #21

### CODE REVIEW

File: src/components/sections/HowItWorks.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const Hiw = () => {
```
Comment (0m3ga13) [inline src/components/sections/HowItWorks.jsx]: I sugegest changing the name of this component for something more representative

**Review by medshk — COMMENTED** [review]
you can name it AppFunctionalityDescription

**Review by 0m3ga13 — APPROVED** [review]

**Comment by 0m3ga13** [comment]
well done Samira!

**Review by 0m3ga13 — APPROVED** [review]

**Review by Emybel — APPROVED** [review]
Great job Samira!

**Pull request merged**